### PR TITLE
remove superfluous wording

### LINKED ISSUE #76: Remove title from nav bar
### Description

I believe the title "**docs.autonity.org**" is unnecessary given it will already be in the url bar, and "documentation" is headlined below. 
image

I propose its removal in order to streamline the look and feel of the documentation site
...

### Rationale

I believe this will helps to declutter the documentation such that only relevant information is displayed. If users wish to return to the home page, they can click on the logo, which has the same functionality as the title (a clickable link to the home page).

The end result would look something like this:
image

### Implementation
Update the `navigation.html` partial (or add one if not already in place) and remove the title shortcode. I recommend this approach because modifying the title so it is an empty string will result in an empty clickable area.

### PR DESCRIPTION
**Pull request by dweng0**
- Remove qualified name from navbar 
- Remove word "Documentation" from LHS navigation
image

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@ baseURL = 'https://docs.autonity.org/'
 enableGitInfo = true
 languageCode = 'en-gb'
 theme = ["docsy", "plausible-hugo"]
-title = 'docs.autonity.org'
+title = 'Documentation'
```
Comment (jay-cousins) [inline config.toml:6]: This could be "Autonity Documentation"?

Comment (jay-cousins) [reply config.toml:6]: Then again, it's clear from the large autonity logo image beneath.

**Review by jay-cousins — APPROVED** [review]
OK

File: config.toml (hunk 1 above)
Comment (dweng0) [reply config.toml:6]: updated:
image

**Pull request merged**